spec/features/applications/show_spec.rb

```diff
@@ -4,14 +4,14 @@
   before(:each) do
     @shelter1 = Shelter.create!(foster_program: true, name: 'Whimsy Whiskers', city: 'Denver', rank: 42)
     @pet1 = @shelter1.pets.create!(name: 'Buddy', breed: 'Golden Retriever', age: 3, adoptable: true)
-    @application1 = Application.create!(name: 'John Doe', address: '123 Elm St, Denver, CO 12392', description: 'Looking for a friendly dog', status: 'Pending')
-    PetApplication.create!(pet: @pet1, application: @application1)
+    @application1 = Application.create!(name: 'John Doe', address: '123 Elm St', city: 'Denver', state: 'CO',
+                                        zip: '12345', description: 'Looking for a friendly dog', status: 'In Progress')
+    # PetApplication.create!(pet: @pet1, application: @application1)
```

Comment:
Let's make sure when we merge in code, we don't leave unnecessary commented out stuff, general housekeeping.

Reply:
good point, thanks for catching that.

Reply:
Good idea! I should of thought about that